src/ray/object_manager/pull_manager.h

```diff
@@ -68,7 +71,15 @@ class PullManager {
   ///
   /// \return True if a pull was cancelled. If there was no pending pull request for the
   /// object this method may return false.
-  bool CancelPull(const ObjectID &object_id);
+  void CancelPull(uint64_t request_id, std::vector<ObjectID> *objects_to_cancel);
```

Comment:
Documentation.

Also, could we return objects_to_cancel as the output directly?